src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -8,7 +8,18 @@ tags:
   - name: Groups
 
 info:
-  description: Open Build Service API
+  description: |
+    The _Open Build Service API_ is a XML API.
+    This documentation is based on the _OpenAPI 3.0.0_ specification and powered by _Swagger_.
+
+    To authenticate, use [HTTP basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) by passing the _Authorization_ header in the form of `Authorization: Basic <credentials>`.
+
+    We don't have any versioning.
+
+    We don't have rate limiting, so be conscious about how you use the API.
+    We do monitor the API, so abuse will lead to a warning, then a ban if it continues.
```

Comment:
Why do you write this so public? We have some rate limiting, but we shouldn't motivate more people to missuse it by promoting it...

Reply:
It's common to have rate limiting in APIs, so we should clearly state what we have.

I don't see any rate limiting in place for the API, at least in Rails application. Is it implemented in the backend?

Reply:
I aggre with @adrianschroeter here. I don't think we need to state this so clearly, it could tempt people to missuse it.

Reply:
The problem we frequently see, like yesterday evening for instance, is that good-will users that don't know the conditions and assume that we care about guarding against this, trigger grave problems for everyone. Just by doing things that make sense. Documenting this will help avoid some of those situations. Documenting the conditions will not turn them from good-will people into people with bad intentions.

On the other hand, if you have someone that comes to the API with bad intentions, not knowing the conditions will not stop them to be bad. It will also not make it in any way harder for them to do so.

Reply:
why we don't have rate limiting in the first place? it could be configured by the administrator. IMO a better solution than document it.

Reply:
I'm all for rate limiting, but this is out of scope for this PR.